### ISSUE TITLE
Value selection for merged cells

### ISSUE THREAD

**Issue by amir-zeldes** [audreyt/ethercalc#484]
If multiple cells containing content are merged, only a single value is selected for the merged cell - the top-left one. This often makes sense, but I would suggest it would be better to follow Excel's behavior, which is:

  * If only one cell is filled, take that value (whatever cell that may be, not only top-left)
  * If multiple cells are filled, take the top/left-most cell that is filled and use that value

So basically, always use content from the top-left-most filled cell in the range being merged for the merge result. I think this make life easier if one happens to want to extend a cell's span upwards or to the left. 

Ideally this behavior would work together with the suggestion in #483 , such that all other cells whose values were not picked then get cleared beneath the merged cell.
